sonar/theme/templates/sonar/page_wiki.html

```diff
@@ -37,6 +39,8 @@
 
 {%- block javascript %}
 {{ super() }}
+<!-- EasyMDE -->
+<script src="https://cdn.jsdelivr.net/npm/easymde/dist/easymde.min.js"></script>
```

Comment:
I suggest to fix the version of easymde to avoid unexpected results after and update.

Reply:
Done